sagar/data/Lab_1sagar.ipynb

```diff
@@ -0,0 +1,1314 @@
+{
```

Comment:
<p>Add a Y label as population</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/suparthghimire/data-analysis-and-visualization-bca-2025/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>